lib/std/debug.zig

```diff
@@ -1760,12 +1760,16 @@ fn handleSegfaultLinux(sig: i32, info: *const os.siginfo_t, ctx_ptr: ?*const c_v
         .netbsd => @ptrToInt(info.info.reason.fault.addr),
         else => unreachable,
     };
-    switch (sig) {
-        os.SIGSEGV => std.debug.warn("Segmentation fault at address 0x{x}\n", .{addr}),
-        os.SIGILL => std.debug.warn("Illegal instruction at address 0x{x}\n", .{addr}),
-        os.SIGBUS => std.debug.warn("Bus error at address 0x{x}\n", .{addr}),
+
+    // Don't use print() as stderr_mutex may still be locked.
```

Comment:
I assume this is what you mean, as you are using *a* function called `print()` :P
```suggestion
    // Don't use std.debug.print() as stderr_mutex may still be locked.
```

Reply:
Yeah, the wording was a bit confusing...